### DISCUSSION TITLE
building bullet3:x64-windows failed with: BUILD_FAILED

### DISCUSSION THREAD

**Discussion by csnorbi11** [microsoft/vcpkg#39208] [Q&A]
I'm trying to install bullet3 with vcpkg by following the tutorial: https://learn.microsoft.com/hu-hu/vcpkg/get_started/get-started-msbuild?pivots=shell-cmd
OS: Win11
IDE: Visual Studio 2022
C++14
Getting these errors:
`Severity Code Description Project File Line Suppression State Details Error building bullet3:x64-windows failed with: BUILD_FAILED Quacke C:\Dev\Quacke\Quacke\EXEC 1
Severity Code Description Project File Line Suppression State Details Error MSB3073 The command ""C:\Dev\vcpkg\vcpkg.exe" install --x-wait-for-lock --triplet "x64-windows" --vcpkg-root "C:\Dev\vcpkg\" "--x-manifest-root=C:\Dev\Quacke\" "--x-install-root=C:\Dev\Quacke\vcpkg_installed\x64-windows\" " exited with code 1. Quacke C:\Dev\vcpkg\scripts\buildsystems\msbuild\vcpkg.targets 183`

**Comment by StarGate-One** [comment]
See #30604 How to report a build failure and open an [issue](https://github.com/microsoft/vcpkg/issues/new?assignees=&labels=category%3Aport-bug&projects=&template=00-report-package-build-failure.yml&title=%5B%3Cport+name%3E%5D+build+failure) instead of using discussions.

**Comment by AugP** [comment]
This may be helpful for diagnosing the issue as well: https://learn.microsoft.com/vcpkg/troubleshoot/build-failures
